src/hlstrafe.cpp

```diff
@@ -2127,6 +2166,14 @@ namespace HLStrafe
 			VecScale<float, 2>(playerCopy.Velocity, (100.0 - (playerCopy.StaminaTime / 1000.0) * 19.0) / 100.0, playerCopy.Velocity);
 
 		CheckVelocity(playerCopy, vars);
+
+		if (curState.Parameters.Type == HLTAS::ConstraintsType::LOOK_AT) {
+			auto playerCopy2 = PlayerData(playerCopy); // trace origin after Strafe() before it happens
```

Comment:
```suggestion
			auto playerCopy2 = PlayerData(playerCopy);
```
don't think this is needed in this form